src/internal/kopia/upload.go

```diff
@@ -976,35 +978,36 @@ func inflateBaseTree(
 	for _, subtreePath := range snap.SubtreePaths {
 		// We're starting from the root directory so don't need it in the path.
 		pathElems := encodeElements(subtreePath.PopFront().Elements()...)
+		ictx := clues.Add(ctx, "subtree_path_elems", pathElems)
```

Comment:
you probably want to use `subtreePath` else you'll get base64 encoded values
```suggestion
		ictx := clues.Add(ctx, "subtree_path_elems", subtreePath)
```